---
title: "scRNA-Seq Analysis | Seurat"
subtitle: "UO BGMP 2018 Cohort"
author: "Kohl Kinning"
date: "`r format(Sys.time(), '%d %B %Y')`"
output:
  html_notebook:
    highlight: pygments
    theme: cosmo
    toc: yes
    toc_float:
      collapsed: yes
editor_options: 
  chunk_output_type: inline
---

# Introduction to single-cell RNA-seq
## Bulk RNA-seq

+ A major breakthrough (replaced microarrays) in the late 00â€™s and has been widely used since
+ Measures the average expression level for each gene across a large population of input cells
+ Useful for comparative transcriptomics, e.g. samples of the same tissue from different species
+ Useful for quantifying expression signatures from ensembles, e.g. in disease studies
+ Insufficient for studying heterogeneous systems, e.g. early development studies, complex tissues (brain)
+ Does not provide insights into the stochastic nature of gene expression

## scRNA-seq
+ A new technology, first publication by (Tang et al. 2009)
+ Did not gain widespread popularity until ~2014 when new protocols and lower sequencing costs made it more accessible
+ Measures the distribution of expression levels for each gene across a population of cells
+ Allows to study new biological questions in which cell-specific changes in transcriptome are important, e.g. cell type identification, heterogeneity of cell responses, stochasticity of gene expression, inference of gene regulatory networks across the cells.

+ Currently there are several different protocols in use, e.g. SMART-seq2 (Picelli et al. 2013), CELL-seq (Hashimshony et al. 2012) and Drop-seq (Macosko et al. 2015)
+ There are also commercial platforms available, including the Fluidigm C1, Wafergen ICELL8 and the 10X Genomics Chromium
+ Several computational analysis methods from bulk RNA-seq can be used
+ In most cases computational analysis requires adaptation of the existing methods or development of new ones

## scRNA-seq workflow
![](./media/RNA-Seq_workflow-5.pdf.jpg)

# Ideal pipeline

# Get started